plugins/flytekit-kf-tensorflow/setup.py

```diff
@@ -5,7 +5,7 @@
 microlib_name = f"flytekitplugins-{PLUGIN_NAME}"
 
 # TODO: Requirements are missing, add them back in later.
-plugin_requires = ["flytekit>=1.3.0b2,<2.0.0"]
+plugin_requires = ["flytekit>=1.6.1", "flyteidl>=1.5.5"]
```

Comment:
any reason for specifying flyteidl here?

Reply:
removed. makes sense to remove them